### ISSUE TITLE
v3 3.1.17 install does not appear to work with a custom manager folder name

### ISSUE THREAD

**Issue by nick0** [evolution-cms/evolution#2030]
(edit: I misread the version as 3.1.17 instead of 3.1.7)

Hello,

Does Evo 3 install work with a custom manager folder name?

I have an Evo 3.1.7 site with a custom name for the manager folder. Both the backend manager and front end web site is working fine with a custom manager name. This is achieved per advice in a different GitHub post by:
```
A config.php file in the root folder with content...
<?php
return [
    'core' => __DIR__ . '/core',
    'manager' => __DIR__ . '/my-custom-mgr-folder',
    'root' => __DIR__
];

and...

A define.php file in the core > custom folder with content...
<?php
define('MGR_DIR', 'my-custom-mgr-folder');

and...

renaming the manager folder in the root (eg to my-custom-mgr-folder)
```

If I try to manually run the Evo 3.1.7 install (by loading the install folder into the root directory and visiting my-domain.com/install/), the install page returns either a page that just says "MGR_DIR is not defined" or at other times a blank white page. Is this is a bug?

If I revert the folder name to manager and edit those two files to simply point to manager, the 3.1.7 install works as expected.

It would be nice if this was tweaked so install can work in Evo 3 with a custom manager folder without having to revert back to plain old manager each time.

I guess this raises a few question with it too...
* Is it still a good idea for security in Evo 3 to rename the manager folder?
* What about the core folder and if so how to achieve?

Many thanks.

**Comment by sashabeep** [comment]
No, 3.x tree does not support custom themes from first versions

**Comment by Seiger** [comment]
<img width="881" height="144" alt="Image" src="https://github.com/user-attachments/assets/36e97339-84e4-4340-a79c-f49b6396edb5" />

<img width="557" height="294" alt="Image" src="https://github.com/user-attachments/assets/2c4957a2-6189-4162-a4c1-f4d728c29c15" />

 Use https://github.com/evolution-cms/installer for custom directory at manager.
